examples/tutorials/Part X - Federated Learning with Secure Aggregation.ipynb

```diff
@@ -0,0 +1,704 @@
+{
```

Comment:
<p>This should be a subtitle with the same font size as "training function"</p>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2178/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>